.changeset/lovely-penguins-rule.md

```diff
@@ -0,0 +1,10 @@
+---
+"hpe-design-tokens": minor
+---
+
+- Removed `color.text.xweak`, `color.icon.xweak`, `color.text.brand`, `color.icon.brand`
```

Comment:
```suggestion
- Removed `color.text.xweak`, `color.icon.xweak`, `color.text.brand`, `color.icon.brand`
- Renamed `enabled` in component tokens to `rest`
```

Reply:
@halocline Adding this suggestion (can elaborate for fully in release notes). But all component tokens (except formField, which will be handled in separate PR) that used "enabled" are now "rest".

The "Removed" are the only ones that were fully deleted and now need to use a different token.